src/pybind/mgr/cephadm/utils.py

```diff
@@ -133,3 +136,10 @@ def file_mode_to_str(mode: int) -> str:
             f'{"x" if (mode >> shift) & 1 else "-"}'
         ) + r
     return r
+
+
+def exponential_backoff(t: int) -> int:
+    if t < 1:  # default non-positive ints to result for t=1
+        return 90
+    # 90s, 6m, 24m, 1h|36m, 6h|24m, 1d|1h|36m, etc.
+    return (4 ** t) * 90
```

Comment:
```suggestion
def exponential_backoff(try_num: int = 0, sleep_interval: int = 90) -> int:
    if try_num < 0:
        return 90
    # 90s, 3m, 6m ..
    return (2 ** tries) * sleep_interval
```
make the interval more frequent and add these as params?